virttest/shared/cfg/guest-os/Windows/Win10/i386.cfg

```diff
@@ -18,7 +18,7 @@
             cdrom_unattended = "images/win10-32/autounattend.iso"
     sysprep:
         unattended_file = unattended/win10-32-autounattend.xml
-    balloon_service, balloon_hotplug, balloon_memhp:
+    balloon_service, balloon_hotplug, balloon_memhp, win_virtio_driver_install_by_installer:
```

Comment:
Is it possible that we have a update case, eg. win_virtio_driver_upgrade_by_installer, which will also call balloon services? :)

Reply:
> Is it possible that we have a update case, eg. win_virtio_driver_upgrade_by_installer, which will also call balloon services? :)

Actually, this case is about "update", In this case, the win_virtio_driver_install_by_installer is used as cfg and the win_virtio_driver_update_by_installer is used as py.